### ISSUE TITLE
Error report after unstaking NET

### ISSUE THREAD

**Issue by robrigo** [greymass/anchor#879]
Running on macOS using the EOS blockchain.

```
TypeError: Cannot read property 'split' of null
    at Function.mapToProps (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:9:1554288)
    at o (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:162044)
    at h (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:163321)
    at file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:163393
    at Object.run (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:159824)
    at i.l.onStateChange (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:160300)
    at Object.notify (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:157143)
    at e.t.notifyNestedSubs (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:156785)
    at i.l.onStateChange (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:160442)
    at Object.notify (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:157143)
```

**Comment by aaroncox** [comment]
Hrm, are you able to reproduce this reliably? I just tried unstaking on a few accounts and couldn't recreate the issue to try and track it down.

**Comment by aaroncox** [comment]
I believe we have this resolved as of 1.0.3. There was an unrelated issue happening in the background with the RAM UI.

**Comment by robrigo** [comment]
Thanks sorry I didn't follow up. )
